core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.ts

```diff
@@ -22,34 +22,33 @@ import { DotMessageService } from '@dotcms/data-access';
 import { DotPagesState, DotPageStore } from '../dot-pages-store/dot-pages.store';
 import { DotActionsMenuEventParams } from '../dot-pages.component';
 
+interface CustomContextMenu extends PrimeNGContextMenu {
+    show(event?: unknown): void;
+}
```

Comment:
If this is a pure primeng component how come we can just change the type?

Reply:
Could be a PrimeNG issue